.circleci/config.yml

```diff
@@ -66,6 +91,17 @@ workflows:
         filters:
           branches:
             only: master
+    - build-image:
+        filters:
+          branches:
+            ignore: master
+    - publish-image:
+        context: quay.io
+        requires:
+        - build-image
+        filters:
+          branches:
+            only: master
```

Comment:
if we require build-images but build-images ignores the branch master will it work?

Reply:
You're right, we have the same mistake in golang-builder.